codeforboston/maple
# Change from update to set bulkwriter

# Summary

#1347 

- This follows the previous issue that arose because this script wouldn't write to the database if fields were the same, which in turn, doesn't call the sync to the Typesense server.
- The work-around for this is to add a Timestamp to each updated testimony, which will force a write every single time, which allows for the sync to occur

# Checklist


# Screenshots

_Add some screenshots highlighting your changes._

# Known issues

_If you've run against limitations or caveats, include them here. Include follow-up issues as well._

# Steps to test/reproduce

_For each feature or bug fix, create a step by step list for how a reviewer can test it out. E.g.:_

1. Run the script by typing `yarn firebase-admin -e local run-script updateDisplayNames.ts` in the terminal.
2. Check to see if the names in Browse Testimony are correct.
3. Go to Firebase emulator and see if the functions log contains any errors and the syncTestimony log.
4. Run the script again and see if the functions log contains the syncTestimony log again.
